.evergreen/auth_oidc/gcp/README.md

```diff
@@ -80,15 +82,19 @@ An example task group would look like:
           - ${DRIVERS_TOOLS}/.evergreen/auth_oidc/gcp/teardown.sh
     - func: other teardown function
   tasks:
-    - oidc-auth-test-gcp-latest
+    - oidc-auth-test-gcp
 ```
 
 ### Environment Variables
 
-Below is an explananion of the environment variables used in the test:
+Below is an explanation of the environment variables used in the test:
 
-- GCPOIDC_AUDIENCE - The value to use in the `TOKEN_AUDIENCE` auth mechanism property.
-- GCPOIDC_ATLAS_URI - The URI of the Atlas cluster configured for OIDC GCP testing.
+- GCPOIDC_AUDIENCE - The value used in the `TOKEN_RESOURCE` auth mechanism property.
 - GCPOIDC_SERVICEACCOUNT - The GCP Service Account to use for GCP access.
 - GCPOIDC_KEYFILE_CONTENT - The base64-encoded GCP keyfile content.
 - GCPOIDC_MACHINE - The GCE machine type to use for the VM.
+- GCPOIDC_ATLAS_USER - The username for admin database access.
+- GCPOIDC_ATLAS_PASSWORD
```

Comment:
Missing explanation for `GCPOIDC_ATLAS_PASSWORD`.

Reply:
Done